.travis.yml

```diff
@@ -69,4 +69,4 @@ jobs:
         - npm install --loglevel=error
       before_script:
       script:
-        - gulp
+        - node node_modules/gulp/bin/gulp.js --require less-loader
```

Comment:
Explanation: Forces `gulp` to be loaded by node properly instead of other path programs, and makes it require the loader for `less`